cranelift/codegen/src/isa/riscv64/inst/args.rs

```diff
@@ -183,6 +202,9 @@ impl Display for AMode {
             &AMode::NominalSPOffset(offset, ..) => {
                 write!(f, "{}(nominal_sp)", offset)
             }
+            &AMode::IncomingArg(offset) => {
+                write!(f, "{}(incoming_arg)", offset)
```

Comment:
Maybe this would be better written as `fp(incoming_arg_area - {offset})`?

Reply:
I'd also take `"-{}(incoming_arg)"` to signal that the offset is subtracted from the base. I don't think it's worth bikeshedding this very much but I think clarifying that it's a subtraction would help with readability.